config/roles.yml

```diff
@@ -255,3 +255,26 @@ point_in_time_full_access:
         - '*'
       allowed_actions:
         - 'manage_point_in_time'
+
+# Allows users to see security analytics detectors and others
+detectors_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/securityanalytics/alerts/get'
```

Comment:
This looks like a new plugin, can we swap the `opendistro` nomenclature for `opensearch`?

Reply:
swapped to opensearch